pyisy/nodes/node.py

```diff
@@ -216,39 +241,107 @@ def parent_node(self):
             return self._nodes.get_by_id(self._parent_nid)
         return None
 
-    """
-    NODE CONTROL COMMANDS.
+    def get_command_value(self, uom, cmd):
```

Comment:
This function is used for fan/climate settings where the ISY expects integer values for the REST command. It translates the friendly names for the commands into the correct integer values for the ISY's UOM.

Example: instead of calling `node.set_fan_speed("1")`, you call `node.set_fan_speed("auto")` and this function knows to send `/rest/nodes/XX XX XX X/cmd/CLIFS/1` -- if an implementation prefers to use the integer values directly, they should just call `node.send_cmd` directly.